### PR TITLE
feat(connection-indicator): implement automatic hiding

### PR DESCRIPTION
**Pull request by virtuacoplenny**

### CODE REVIEW

File: react/features/connection-indicator/components/ConnectionIndicator.js [hunk 1]
```diff
@@ -6,44 +7,22 @@ import { ConnectionStatsTable } from '../../connection-stats';
...
-];
+const INDICATOR_DISPLAY_THRESHOLD = 64;
+const CONNECTION_TO_COLOR_MAP = {
+    0: 'crimson',
+    30: 'indianred',
+    47: 'yellow',
+    [INDICATOR_DISPLAY_THRESHOLD]: 'palegreen',
+    81: 'springgreen',
+    98: 'springgreen'
+};
```
Comment (virtuacoplenny) [inline react/features/connection-indicator/components/ConnectionIndicator.js]: I didn't know what colors to use so I just picked some from a red-to-green scale. I also didn't know at what percentages to trigger different behaviors, so I copied the original implementation--https://github.com/jitsi/jitsi-meet/pull/1012.

File: react/features/connection-indicator/components/ConnectionIndicator.js [hunk 2]
```diff
@@ -6,44 +7,42 @@ import { ConnectionStatsTable } from '../../connection-stats';
...
 
-    // Note: we never show 0 bars.
-];
+/**
+ * A map of connection quality percentages and corresponding colors to display
+ * when the current connection quality meets or exceeds the percentage.
+ *
+ * @type {Object}
+ */
+const CONNECTION_TO_COLOR_MAP = {
```
Comment (virtuacoplenny) [inline react/features/connection-indicator/components/ConnectionIndicator.js]: I didn't know what colors to use so I just picked some from a red-to-green scale. I also didn't know at what percentages to trigger different behaviors, so I copied the original implementation--https://github.com/jitsi/jitsi-meet/pull/1012.

File: react/features/invite/components/DialInNumbersForm.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import Button from '@atlaskit/button';
 import { StatelessDropdownMenu } from '@atlaskit/dropdown-menu';
 import { FieldText } from '@atlaskit/field-text';
-import ExpandIcon from '@atlaskit/icon/glyph/expand';
+import ExpandIcon from '@atlaskit/icon/glyph/editor/expand';
```
Comment (yanas) [inline react/features/invite/components/DialInNumbersForm.js]: This icon hasn't been introduced in this PR and it's my fault that I have missed it when reviewing, but we need to stick to the font icons for now for the reason mentioned above.

**Comment by virtuacoplenny** [comment]
Closing because the requested changes will be easier to do from scratch.

**Comment by virtuacoplenny** [comment]
Redone in https://github.com/jitsi/jitsi-meet/pull/2009. All comments have been addressed, except for using the atlaskit vid-connection-circle svg as an icon.

**Pull request closed without merging**